event_model/tests/test_auth.py

```diff
@@ -1,7 +1,8 @@
-import event_model
 import pytest
 from jsonschema.exceptions import ValidationError
 
+import event_model
+
```

Comment:
Please restore the non-isorted versions of test files file in this commit, then apply in the blackening+isorting commit